### PR TITLE
New command to set the temperature LUT.

### PR DESCRIPTION
**Pull request by teweitsai**

### CODE REVIEW

File: python/lsst/ts/m2/csc.py [hunk 1]
```diff
@@ -122,6 +123,13 @@ def __init__(
         simulation_mode: int = 0,
         verbose: bool = False,
     ) -> None:
+        # This is to keep the backward compatibility of ts_xml v22.0.0 that
+        # does not have the 'enableLutTemperature' command defined in xml.
+        # TODO: Remove this after ts_xml v22.1.0.
+        component_info = ComponentInfo("MTM2", "sal")
+        if "cmd_enableLutTemperature" in component_info.topics:
```
Comment (pkubanek) [inline python/lsst/ts/m2/csc.py:130]: I don't like this coding style except in cases it's really needed. Do you really need to check if the command is there? It's problem of the deployment policy to not deploy apps not matching XML schema. The default behavior if the command isn't in topic would be an error, which tells you more - either roll back old application, or wait for XML update. This just add complexity to the code when it's not needed.

File: tests/test_csc.py [hunk 2]
```diff
@@ -1058,10 +1079,11 @@ async def test_switchForceBalanceSystem(self) -> None:
                 flush=True, timeout=STD_TIMEOUT
             )
 
-            # LUT gravity and temperature absolute values should be larger
-            # than zero
+            # LUT gravity absolute value should be larger than zero
             self.assertTrue(np.all(np.abs(axial_forces.lutGravity) > 0.0))
-            self.assertTrue(np.all(np.abs(axial_forces.lutTemperature) > 0.0))
```
Comment (pkubanek) [inline tests/test_csc.py:1064]: You might want to add max value as well. The temperature offsets shall be a few N, so something like:
```
self.assertTrue(np.all(np.abs(axial_forces.lutTemperature) > 0.0))
self.assertTrue(np.all(np.abs(axial_forces.lutTemperature) < 20.0))
```

or even:
```
self.assertTrue(np.all(np.abs(10 > axial_forces.lutTemperature) > 0.0))
```

**Review by pkubanek — APPROVED** [review]

File: python/lsst/ts/m2/csc.py (hunk 1 above)
Comment (teweitsai) [reply python/lsst/ts/m2/csc.py:130]: I do not like this as well and had suggested Tiago to have a better way to deal with the backward compatibility in the future's Kafka version. Before the `ts_xml` and `ts_salobj` have the further support, we might need to live with this workaround for a while. Thanks!

File: tests/test_csc.py (hunk 2 above)
Comment (teweitsai) [reply tests/test_csc.py:1064]: The line change of `1064` was removed actually. Thanks!

**Pull request merged**